### PR TITLE
Adds consoledot pages best practices section

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-26876

Rest of the content will follow. This is a base setup.

### CODE REVIEW

**Review by karelhala — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM quay.io/cloudservices/releaser:875ee35
+FROM quay.io/cloudservices/releaser:24ff556
```
Comment (karelhala) [inline Dockerfile:1]:
```suggestion
FROM quay.io/cloudservices/releaser:6969d44
```

**Comment by Hyperkid123** [comment]
Don't merge this PR. It may break local development if your setup does not contain correct binaries. A docker image for development is incoming.

**Pull request merged**